src/query.js

```diff
@@ -15,6 +15,7 @@ function query (data, params = {}) {
   // TODO clean up this series of if statements
   if (data.filtersApplied && data.filtersApplied.geometry) delete params.geometry
   if ((data.filtersApplied && data.filtersApplied.where) || params.where === '1=1') delete params.where
+  if (data.filtersApplied && data.filtersApplied.offset) { params.resultOffset = 0; }
```

Comment:
please follow the style above of using no brackets and `delete`ing the applied parameter.